hack/make/resources/docker.exe.manifest

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Can we rename this directory to be something more like `hack/make/.windows-resources`?